nodewatcher/modules/frontend/ip_space/static/css/tipsy.css

```diff
@@ -0,0 +1,129 @@
+/* Author of tipsy: https://github.com/jaz303
+   tipsy repository: https://github.com/jaz303/tipsy
+   modified by: https://bl.ocks.org/ilyabo
+   copied from: http://bl.ocks.org/ilyabo/1373263
```

Comment:
Please create a `wlanslovenija` branch in this repo (from `master` or `v2` branch, whichever is closer to http://bl.ocks.org/ilyabo/1373263): https://github.com/wlanslovenija/tipsy

Then, in that branch, apply changes from http://bl.ocks.org/ilyabo/1373263.

Then, use git submodule to pull that branch in here.